docs/configuration.md

```diff
@@ -13,3 +13,11 @@ Certain features of the React application import variables from an [AppConfig](h
 ### Custom Fonts & Styles
 Fonts and css variables can be customized by modifying [fonts-custom.scss](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_fonts-custom.scss) and
 [variables-custom.scss](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_variables-custom.scss).
+
+### Examples
+
+#### 1) To add a custom logo you need to do the following:
```

Comment:
nit: `Add a custom logo`. Succinct section titles can make doc easier to scan for folks looking to find a single specific piece of info.

Reply:
Agree, will fix